app/views/home/index.html.erb

```diff
@@ -0,0 +1,41 @@
+<style>
```

Comment:
Agreed. I was waiting for the actual css to arrive... but I can go ahead and just create one.